lang/java/reef-applications/reef-vortex/README.md

```diff
@@ -0,0 +1,17 @@
+Vortex
+========================
+Vortex is a runtime for low-priority jobs to effectively cope 
```

Comment:
Also, can we have a summary of this text in the JavaDoc for the package? That way, it becomes discoverable on our website.